### PR TITLE
Fix click area on social buttons

### PR DESCRIPTION
**Pull request by hopemanryan**
### Type of change

- [ ] Improvement
- [x] Bugfix
- [ ] Refactor
- [ ] New feature

### Size of change

- [x] Small (<3 files)
- [ ] Medium (new file(s) or >3 files)
- [ ] Large (>10 files or major change)

### Description of changes
Override the footer styles for social icons 
### QA Procedure

Hover on the icons and see the range of the click area by looking at the url chrome shows 

### Screenshots (if visual changes were made)

Insert screenshots here
image

image

### CODE REVIEW

**Comment by Urook** [comment]
I don't know the docs very well. Can you add both the before and after images?

**Comment by hopemanryan** [comment]
> I don't know the docs very well. Can you add both the before and after images?

Added the image

**Comment by Urook** [comment]
> > I don't know the docs very well. Can you add both the before and after images?
> 
> Added the image @Urook 
> 

I can't spot the difference😅. Perhaps you could tag the whole team

File: website/static/css/custom.css [hunk 1]
```diff
@@ -1080,3 +1080,21 @@ footer .social img {
...
+footer .social  a {
+    display: flex !important;
+    justify-content: center;
+    align-content: center;
+    padding: unset !important;
+    margin: unset !important;
+}
+footer .social  a img {
+    margin: unset;
+    transform: scale(1.3);
```
Comment (ron-rookout) [inline website/static/css/custom.css:1099]: The images don't look quite the same size, why scale them and not give them a fixed size?

Comment (hopemanryan) [reply website/static/css/custom.css:1099]: the margin gave it size but it also what caused the overflow on the a tag 
this was a way to keep the image size as before with out changing the image it self

Comment (hopemanryan) [reply website/static/css/custom.css:1099]: image
side by side 
looks very very close to me

Comment (hopemanryan) [reply website/static/css/custom.css:1099]: @ron-rookout

**Pull request merged**